SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialGameInstance.cpp

```diff
@@ -67,7 +67,8 @@ bool USpatialGameInstance::HasSpatialNetDriver() const
 
 void USpatialGameInstance::CreateNewSpatialWorkerConnection()
 {
-	SpatialConnection = NewObject<USpatialWorkerConnection>();
+	SpatialConnection = NewObject<USpatialWorkerConnection>(this);
```

Comment:
Do we want to make the GameInstance the outer?

Reply:
I always explicitly add outers to highlight who is supposed to be owning the object. Seems like that's not the case elsewhere in the codebase though so happy to change